src/components/navbar/NavbarUser.js

```diff
@@ -76,6 +80,12 @@ const NavbarUser = props => {
                   <MenuItem className={classes.dropdownHeader}>
                     {user.fullName}
                   </MenuItem>
+                  <MenuItem
```

Comment:
It's not an in your face UX solution, but at least it's possible now! 🙏 for putting this together!